openapi-generator-annotations/lib/src/openapi_generator_annotations_base.dart

```diff
@@ -152,6 +172,164 @@ class Openapi {
   });
 }
 
+/// Provides the input spec file to be used.
+///
+/// Provides the location of the input spec file to be used by the generator.
+/// Includes the option to use the default json or yaml paths.
+class InputSpec {
+  final String path;
+  final bool defaultYaml;
```

Comment:
I don't think there is a need for this flag. You can just check for openapi.{yaml|yml|json} unless you are using it for something else

Reply:
I intended on it being used by the `GeneratorArguments` to provide the default openapi spec file. But it looks like I never made those changes. I can that tweaked.